src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala

```diff
@@ -99,16 +94,38 @@ class RoutesPostgresRepo @Inject() (
   override def selectByName(name: String): DatabasePublisher[RouteRow] = {
     logger.debug(s"selectByName $name")
 
-    val q = for {
+    val q = (for {
       routeRow <- routesTable
       if routeRow.name === name && routeRow.deletedAt.isEmpty
-    } yield routeRow
+    } yield routeRow)
 
     db.stream {
       q.result
     }
   }
 
+  override def selectLatestActiveRoutesPerName(currentTime: LocalDateTime): DatabasePublisher[RouteRow] = {
+    logger.debug("select latest routes per name")
+
+    val q = (for {
+      routeRow <- routesTable
+      if routeRow.deletedAt.isEmpty && routeRow.activateAt < currentTime
```

Comment:
Is it possible to deactivate a route? Or can I just `create -> activate -> delete` with no way back?